### PR TITLE
[Fixes #74] Add support for netcoreapp1.0 on DependenciesPackager

### LINKED ISSUE #74: Make dependencies packager cross-platform

### PR DESCRIPTION
**Pull request by javiercn**

### CODE REVIEW

**Comment by javiercn** [comment]
@pranavkm

File: src/DependenciesPackager/Program.cs [hunk 1]
```diff
@@ -606,6 +657,11 @@ private ProjectContext CreateProjectContext(Project project, NuGetFramework fram
             }
         }
 
+        private string GetPlatform(string moniker)
```
Comment (pranavkm) [inline src/DependenciesPackager/Program.cs]: What's this doing here? Could you add a comment?

File: src/DependenciesPackager/Program.cs [hunk 2]
```diff
@@ -561,13 +610,14 @@ private ProjectContext CreateProjectContext(Project project, NuGetFramework fram
                 throw new InvalidOperationException("Couldn't find the dependency 'Microsoft.NETCore.Runtime.CoreCLR'");
             }
 
+            var crossGen = CrossGenFiles[GetPlatform(moniker)].CrossGen;
```
Comment (pranavkm) [inline src/DependenciesPackager/Program.cs]: `crossGenExecutable`

**Comment by pranavkm** [comment]
:shipit:

**Pull request merged**